src/devhub/index.html

```diff
@@ -49,15 +54,15 @@ <h3>Links</h3>
         <h3>Main branch fuzzes</h3>
         <table>
           <tr>
-            <td class="ok" style="padding-right: 16px;">ok</td>
+            <td class="ok" style="padding-right: 16px;" title="This many fuzzes didn't have any failure.">ok</td>
```

Comment:
Can we add some concept of time and also which branch(es) this relates to? Or remove confusion around these?

Reply:
Branch is in the `h3`: `<h3>Main branch fuzzes</h3>`. 

For time, this is under-specified, but it's over our entire fuzzing database (our database is constant-size, so we trim old entries)